NFe.Utils/Email/ConfiguracaoEmail.cs

```diff
@@ -109,5 +109,10 @@ private ConfiguracaoEmail()
         /// Se for marcado como false, a aplicação irá esperar pelo envio
         /// </summary>
         public bool Assincrono { get; set; }
+
+        /// <summary>
+        /// Nome do remetente do Email.
+        /// </summary>
+        public string Nome { get; set; }
```

Comment:
Esse nome e de quem enviou o email correto?
Se sim acho que poderia melhorar NomeRemetente ou algo assim 
Porque lendo aqui na hora pensei tem o nome para destinatário no método AdicionarDestinatario
vendo aqui pelo que entendi e o nome de quem esta enviando o email (não cheguei a testar)
se for isso mesmo acho que poderíamos melhorar somente um pouco nisso que falei

Reply:
oi @robertorp  coloquei somente Nome porque a propriedade Email tá assim:

/// <summary>
/// Endereço de e-mail do remetente
/// </summary>
public string Email { get; set; } 

daí segui a mesma linha.....mas qqr coisa eu mudo de cá e subo de novo adicionado ao pull, sem problemas.

Reply:
Verdade, sendo assim vamos deixar assim. 
Fica padronizado. 
Não tinha reparado nisso.
Obrigado @gui-abrantes